src/ui_ng/src/app/app.module.ts

```diff
@@ -49,6 +50,12 @@ export function getCurrentLanguage(translateService: TranslateService) {
         AccountModule,
         HarborRoutingModule,
         ConfigurationModule,
+        FormsModule,
```

Comment:
I think those two modules importing should be moved to share module

Reply:
I tried to import them on shared module at first time but it seems not work.